package.json

```diff
@@ -16,8 +16,8 @@
     "build:dev": "webpack --mode development --https false",
     "build-dev": "webpack --mode development --https false && echo . && echo . && echo . && echo Please use 'build:dev' instead of 'build-dev'.",
     "dev-server": "webpack-dev-server --mode development",
-    "lint": "eslint '**/*.{js,jsx}'",
-    "lint:fix": "eslint '**/*.{js,jsx}' --fix",
+    "lint": "eslint \"src/**/*.{js,jsx}\"",
```

Comment:
Can't we just write it as `"lint": "eslint src/**/*.{js,jsx}",`? I remember it worked well like that.

Reply:
Yeah sure! I had tried this pattern and it worked, So I will update my PR soon